src/main/java/com/zontik/groshiky/service/TransactionService.java

```diff
@@ -30,7 +30,17 @@ public Transaction addTransaction(Transaction transaction, Account account) {
         return transactionRepository.save(transaction);
     }
 
+    @Override
+    public void deleteById(Integer id) {
```

Comment:
Methods that works with changing database entities should work in transaction (I'm talking about database transactions). Add DB transaction to this method and to method, that adds new Transaction entity